packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -683,6 +711,11 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
   private _getAxisData = (yAxisData: IAxisData) => {
     if (yAxisData && yAxisData.yAxisDomainValues.length) {
       const { yAxisDomainValues: domainValue } = yAxisData;
+      const domainRange =
```

Comment:
domain is constructed using datapoints only. Move this logic to datapoints only.

Reply:
done